### PR TITLE
Add the sync pings view job on Airflow

### PR DESCRIPTION
**Pull request by Dexterp37**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-airflow/53)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: dags/sync_view.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+from airflow.operators import BashOperator
+
+default_args = {
+    'owner': 'markh@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2016, 12, 8),
```
Comment (vitillo) [inline dags/sync_view.py]: Do you really mean to use the 8th of December as start date?

**Comment by Dexterp37** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[dags/sync_view.py, line 9 at r1](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/53#-KTKICZrJdpbZiaJQcpk-r1-9:-KTKNiA9RtlQI1sYz0am:bhmjcz3) ([raw file](https://github.com/mozilla/telemetry-airflow/blob/d9155988478782d8b90d94db0f907a781ea113c4/dags/sync_view.py#L9)):*
<details><summary><i>Previously, vitillo (Roberto Agostino Vitillo) wrote…</i></summary>
> Do you really mean to use the 8th of December as start date?
</details>
Whops, no. Fixed that! Let's not merge it yet as I'm waiting to hear back from Mark Hammond. Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/53)*

**Comment by Dexterp37** [comment]
Switched the job to run daily as per Mark's request. r?@itillo

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/53#-:-KTOPq4tOlaGbZlTCIbu:b-p0adls)*

**Comment by Dexterp37** [comment]
Well, I meant r?@vitillo

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/53#-:-KTOPyJxbxkHmBOtDNvQ:bdm1dlv)*

File: dags/sync_view.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('sync_view', default_args=default_args, schedule_interval='@daily')
+
+t0 = EMRSparkOperator(task_id="sync_view",
+                      job_name="Sync Pings View",
+                      execution_timeout=timedelta(hours=10),
+                      release_label="emr-5.0.0",
+                      instance_count=10,
```
Comment (vitillo) [inline dags/sync_view.py]: Do we really need 10 instances? How long does the job take to run on a single instance for a single day?

Comment (Dexterp37) [inline dags/sync_view.py]: ?@mhammond , as he is in a better position to answer that :)

**Comment by vitillo** [comment]
r+

**Comment by vitillo** [comment]
Deployed.

**Pull request merged**